src/schema/tags.ts

```diff
@@ -192,13 +206,17 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         .orderBy('value', 'ASC')
         .execute();
 
+      if (shuffle) {
+        hits.sort(() => Math.random() - 0.5);
+      }
```

Comment:
Hmm while implementation is fine, I would ask is it better to randomize on the frontend? Feels like a niche thing which would not much sense to do on API because you would get random result each time, while on frontend you can actually control the seed and when you randomize. Does that make sense?

Reply:
I am fine with either. Thoughts @rebelchris ?

Reply:
Moved to frontend:
https://github.com/dailydotdev/apps/pull/3617/commits/cd68200837abfb16e434d29749f1250c58d603fc

Kept the recommendation the same as they ranked by probability and that seemed odd to shuffle.

Reply:
Should I close this then?